plugins/available/blesh.plugin.bash

```diff
@@ -11,6 +11,10 @@ _bash_it_ble_path=${XDG_DATA_HOME:-$HOME/.local/share}/blesh/ble.sh
 if [[ -f $_bash_it_ble_path ]]; then
 	# shellcheck disable=1090
 	source "$_bash_it_ble_path" --attach=prompt
+	if _bash-it-component-item-is-enabled plugin fzf; then
+		ble-import integration/fzf-key-bindings
```

Comment:
Is there a specific reason that the option `-d` (delayed loading) is dropped for `fzf-key-bindings`, but not for `fzf-completion`?

Reply:
Whether to specify `-d` or not is up to the user actually. Both cases with and without `-d` work, but it is a bit weird that one `fzf` module is loaded without `-d`, while the other is loaded with `-d`. I recommend specifying `-d` because the `fzf` integration is not used by the prompt, so the initialization can be scheduled after showing the prompt. This way, the startup time until the first prompt becomes shorter.